CMakeLists.txt

```diff
@@ -115,6 +115,10 @@ find_program( SED NAMES sed gsed )
 ##################   CPack, checks, ... in external modules   ##################
 ################################################################################
 
+include( GetTriple )
```

Comment:
In autotools they are called [`TypeTriplets`](https://sourceforge.net/p/mingw-w64/wiki2/TypeTriplets/). Other cmake implementations call this [`GetTriplet.cmake`](https://opensource.apple.com/source/clang/clang-421.0.57/src/projects/libcxx/cmake/Modules/GetTriple.cmake), or `Triplet.cmake`[[1]](https://github.com/roxlu/SDP/blob/master/build/Triplet.cmake) [[2]](https://github.com/roxlu/video_streamer/blob/master/tools/cmake/Triplet.cmake). As this is specifically for the autotools triplets, I think the name triplet should be contained.